Plugin~/WebRTCPlugin/WebRTCPlugin.cpp

```diff
@@ -1346,6 +1346,11 @@ extern "C"
         dataChannelObj->Send(msg);
     }
 
+    UNITY_INTERFACE_EXPORT void DataChannelSendPtr(DataChannelObject* dataChannelObj, const byte* msg, int len)
```

Comment:
It looks like `DataChannelSendBinary` function same.

Reply:
It is, I forgot about [DllImportAttribute.EntryPoint](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.dllimportattribute.entrypoint?view=net-5.0).  Fixed in next commit.